Preloader Functionality Added

shreya024 opened the pull request:
I have added Preloader functionality.
Referring to issue number 
fixes #17

DhairyaBahl commented:
@shreya024 Well done but That continuous scale on the logo is not looking as expected. How about just a logo in the middle of the screen. Remove the `HASH DEFINE` that you wrote below the image. and add a horizontal loading bar below the image. Hope you are getting my point.

shreya024 commented:
> @shreya024 Well done but That continuous scale on the logo is not looking as expected. How about just a logo in the middle of the screen. Remove the `HASH DEFINE` that you wrote below the image. and add a horizontal loading bar below the image. Hope you are getting my point.

Kindly check now .I have removed the text and added a horizontal loading bar but kept the animation.

vedant-jain03 commented on src/Components/Preloader/Preloader.scss:
```diff
@@ -49,24 +57,24 @@
+    position: relative;
+    padding: 0 5px;
+    display: flex;
+    height: 40px;
```
Kindly make `height: 20px`

vedant-jain03 commented:
Please uncommit `package-lock.json` file.

vedant-jain03 commented:
Good work @shreya024
